<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 255, 127);
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>23</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Formulário de Cadastro </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixaDescricao">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixaDataIni">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>150</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2026</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>3</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QDateEdit" name="caixaDataTer">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>150</y>
        <width>241</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
      <property name="maximumDate">
       <date>
        <year>2026</year>
        <month>12</month>
        <day>31</day>
       </date>
      </property>
      <property name="minimumDate">
       <date>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </date>
      </property>
     </widget>
     <widget class="QLineEdit" name="caixaRespo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="caixaPriori">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>230</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="caixaObs">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>320</y>
        <width>551</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Descrição da Tarefa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data de Inicio</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>120</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Data de Término</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Responsável</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>200</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Tipo de Prioridade</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>290</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Observações</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnCadastrar">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>540</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
      <property name="text">
       <string>CADASTRAR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLimpar">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>540</y>
        <width>75</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 2px;</string>
      </property>
      <property name="text">
       <string>LIMPAR</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
